components/dashboard/TextArea/index.tsx

```diff
@@ -49,14 +51,33 @@ const TextArea = ({ shiftRight, setShiftRight }: TextAreaProps) => {
       ).withConverter(notesConverter)
   );
 
+  const addToStore = async (key: string, value: IDBObject) => {
+    await set(key, value);
+  };
+
+  const getFromStore = async (
```

Comment:
We can export the other getFromStore and use that here. Currently we're breaking DRY rule.

We could actually also setup a hook for idb interaction.

Reply:
Yes, I overlooked that. Thanks

Reply:
Implemented an useIDB hook :)